modules/digitaluniverse/rendering/renderablebillboardscloud.cpp

```diff
@@ -698,14 +698,14 @@ void RenderableBillboardsCloud::render(const RenderData& data, RendererTasks&) {
     glm::vec3 orthoRight = glm::normalize(glm::vec3(worldToModelTransform * glm::vec4(right, 0.0)));
     glm::vec3 orthoUp = glm::normalize(glm::vec3(worldToModelTransform * glm::vec4(up, 0.0)));
 
-
+      
```

Comment:
Some trailing space inserted into this file

Reply:
Was due to a missing merge with `master`.. I had removed the trailing spaces on `master`